packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -349,6 +376,38 @@ export default class SrDisks extends Component {
     },
   ]
 
+  _getBaseCopiesByVdi = createSelector(
+    () => this.props.vdis,
+    () => this.props.unmanagedVdis,
+    (vdis, unmanagedVdis) => {
+      const vdisByBaseCopy = {}
```

Comment:
baseCopiesByVdi

Reply:
It's a list of VDIs indexed by the corresponding base copy's ID. So `vdisByBaseCopy` is correct.